packages/markdown/processor/parse/magic-block-parser.js

```diff
@@ -51,7 +51,10 @@ function tokenize(eat, value) {
           },
         },
       }));
-      if (children.length === 1 && children[0].name) return eat(match)(WrapPinnedBlocks(children[0], json));
+      if (children.length === 1) {
+        if (!children[0].value) return eat(match); // skip empty code tabs
```

Comment:
for if, like, when a user drags in a code tab block to the end of a long doc, and then forgets about it but saves anyways, right? now that won't break shit.